agent/src/main/java/com/appland/appmap/transform/instrumentation/BBTransformer.java

```diff
@@ -0,0 +1,71 @@
+package com.appland.appmap.transform.instrumentation;
+
+import java.lang.instrument.Instrumentation;
+import java.security.ProtectionDomain;
+
+import org.tinylog.TaggedLogger;
+
+import com.appland.appmap.config.AppMapConfig;
+import com.appland.appmap.transform.annotations.AppMapAppMethod;
+import com.appland.appmap.transform.annotations.AppMapInstrumented;
+
+import net.bytebuddy.agent.builder.AgentBuilder;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.dynamic.DynamicType.Builder;
+import net.bytebuddy.jar.asm.Label;
+import net.bytebuddy.jar.asm.MethodVisitor;
+import net.bytebuddy.jar.asm.Opcodes;
+import net.bytebuddy.matcher.ElementMatchers;
+import net.bytebuddy.utility.JavaModule;
+
+public class BBTransformer implements AgentBuilder.Transformer {
+  private static final TaggedLogger logger = AppMapConfig.getLogger(null);
+
+  public static void installOn(Instrumentation inst) {
+    new AgentBuilder.Default()
+        .type(ElementMatchers.isAnnotatedWith(AppMapInstrumented.class))
+        .transform(new BBTransformer())
+        .installOn(inst);
+  }
+
+  private BBTransformer() {}
+
+  static class LineNumberExtractorMethodVisitor extends MethodVisitor {
```

Comment:
I just noticed that this got left in by mistake. I'll take it out before merging.